R/vsc.R

```diff
@@ -323,7 +323,7 @@ if (show_view) {
     list(columns = columns, data = data)
   }
 
-  show_dataview <- function(x, title,
+  show_dataview <- function(x, title, .UUID = NULL,
```

Comment:
Is there a reason to use `.UUID` instead of `UUID` here? I don't see a potential conflict or misuse here.

Also, it looks a bit inconsistent with the lower-case argument code-style in the rest of R code.

Would you mind changing all `UUID` into `uuid`?

Reply:
I was using `.UUID` in case of accidental argument matching, but it is unlikely that something would go wrong there. Lowercase makes sense too!